src/components/commons/Footer/Footer.jsx

```diff
@@ -1,14 +1,18 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useContext } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useHistory } from 'react-router'
-import { REGISTER } from 'routing/routes'
+import { PROFILE_USER, REGISTER } from 'routing/routes'
 import SocialMediaButtons from 'components/commons/SocialMediaButtons'
 import Button from 'components/commons/Button'
 import styles from './footer.scss'
+import UserContext from 'Context/UserContext'
 
 const Footer = () => {
   const history = useHistory()
   const { t } = useTranslation('landingPage')
+  const { authStore } = useContext(UserContext);
+
+  const handleGoToProfile = useCallback(() => history.push(`${PROFILE_USER}/${authStore.user._id}`), [])
 
   const handleGoToRegister = useCallback(() => {
```

Comment:
It's okay for you to change this functions for these? 😃
````
  const goTo = useCallback(() => {
    if (authStore.isLogin) return history.push(`${PROFILE_USER}/${authStore.user._id}`)
    history.push(REGISTER)
  }, [authStore.isLogin])

  const formatText = useCallback(() => {
    if (authStore.isLogin) return t('navbar:myProfile')
    t('common:signUp')
  }, [authStore.isLogin])
````

Reply:
looks cleaner, I've changed it